assets/store/Conversation.js

```diff
@@ -53,6 +53,7 @@ export default class Conversation extends Reactive {
 
     if (params.conversation_id) {
       this.prop('persist', 'wantNotifications', this.is_private, {key: params.conversation_id +  ':wantNotifications'});
+      this.prop('rw', 'userInput', '');
```

Comment:
If you want to make it persistent (meaning the input value survives a browser refresh), you can do something like this:

```
this.prop('rw', 'userInput', '', {key: ['userInput', params.connection_id, params.conversation_id].join(':')});
```

"key" is the value key used to store the value in [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage). It will default to the prop name, but you have to override it to make it unique per Conversation object.